Samples/CayenneDecoderTest/CayenneTest.cs

```diff
@@ -0,0 +1,189 @@
+using System;
```

Comment:
Love the tests. I am missing a e2e test that will verify that (string payload, byte fport) => returns a correct json that will be picked by the network server.